docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -279,212 +216,237 @@ server, DHCP server, RADIUS server) using:
+Next we configure a switch so that it integrates with PacketFence using 802.1X. In our example, we will use a Cisco Catalyst 2960 access switch and its IP address will be 172.21.2.3. Our PacketFence's server IP address will be 172.20.100.2 - you will need to adjust this according to your environment.
 
-In this context, PacketFence runs the authentication server (a FreeRADIUS instance) and will return the appropriate VLAN to the switch. A module that integrates in FreeRADIUS does a remote call to the PacketFence server to obtain that information. More and more devices have 802.1X supplicant which makes this approach more and more popular.
+Connect to that switch over SSH as an admin.
 
-MAC Authentication is a new mechanism introduced by some switch vendor to handle the cases where a 802.1X supplicant does not exist. Different vendors have different names for it. Cisco calls it MAC Authentication Bypass (MAB), Juniper calls it MAC RADIUS, Extreme Networks calls it Netlogin, etc. After a timeout period, the switch will stop trying to perform 802.1X and will fallback to MAC Authentication. It has the advantage of using the same approach as 802.1X except that the MAC address is sent instead of the user name and there is no end-to-end EAP conversation (no strong authentication). Using MAC Authentication, devices like network printer or non-802.1X capable IP Phones can still gain access to the network and the right VLAN.
+Enable 802.1X
+^^^^^^^^^^^^^
+As a first configuration step, you need to enable 802.1X globally on the switch.  To do so, use the following:
 
-Wireless: 802.1X + MAC authentication
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   dot1x system-auth-control
 
-Wireless 802.1X works like wired 802.1X and MAC authentication is the same as wired MAC Authentication. Where things change is that the 802.1X is used to setup the security keys for encrypted communication (WPA2-Enterprise) while MAC authentication is only used to authorize (allow or disallow) a MAC on the wireless network.
+Configure AAA
+^^^^^^^^^^^^^
+The next step is to configure AAA so it will use your newly created PacketFence server.  Make sure you replace the PF_MANAGEMENT_IP variable with your actual PacketFence management IP (172.20.100.2 in our example) in the following commands:
+
+   aaa new-model
+   aaa group server radius packetfence
+    server PF_MANAGEMENT_IP auth-port 1812 acct-port 1813
+   aaa authentication login default local
+   aaa authentication dot1x default group packetfence
+   aaa authorization network default group packetfence
+   radius-server host PF_MANAGEMENT_IP auth-port 1812 acct-port 1813 timeout 2 key useStrongerSecret
+   radius-server vsa send authentication
```

Comment:
I don't know if its wanted, but the CoA configuration is not here.

Reply:
It's added later for the web auth thing. We don't need to do CoA right from the start with PacketFence configured as a simple RADIUS server.

Reply:
Alright, fine by me